content/rationale.md

```diff
@@ -0,0 +1,85 @@
+title: Why complate?
+
+Why complate?
+=============
+
+## Abstraction
+
+When we are developing our frontend components, we can use complate markup
```

Comment:
another FNDism: "front-end" (throughout, same for "back{ ,-}end")

> The term “front-end” is correct when used as a compound adjective, and the term “front end” is correct when used as a noun.

https://css-tricks.com/poll-results-front-end-front-end/